apps/xplat-v9/StrictDomDemo.tsx

```diff
@@ -17,11 +18,11 @@ const useBaseClassName = makeResetStyles({
 const useClassNames = makeStyles({
   root: {
     ...shorthands.border('2px', 'solid', 'red'),
-    color: 'pink',
-    backgroundColor: '#333',
+    color: '#222',
```

Comment:
Does strict DOM not support CSS color names?

Reply:
It does, I was just fiddling with this to make it look a little less weird (no technical reason for this change). These styles are going to be removed soon in the branch -- we're going to add theme/token support soon, and we'll use tokens at that point.